runtime-modules/storage/src/data_directory.rs

```diff
@@ -358,6 +368,30 @@ decl_module! {
             Self::deposit_event(RawEvent::ContentAdded(content, owner));
         }
 
+        /// Remove the content from the system.
+        #[weight = 10_000_000] // TODO: adjust weight
+        pub fn remove_content(
+            origin,
+            owner: StorageObjectOwner<MemberId<T>, ChannelId<T>, DAOId<T>>,
+            content_ids: Vec<ContentId<T>>
+        ) {
+
+            // Ensure given origin can perform operation under specific storage object owner
+            Self::ensure_storage_object_owner_origin(origin, &owner)?;
```

Comment:
I'm not seeing an implementation of this function `ensure_storage_object_owner_origin()`. I assume this going to come from a trait to be implemented in the runtime module that can validate origin vs owner ?

Reply:
It`s here. https://github.com/Joystream/joystream/blob/sumer/runtime-modules/storage/src/data_directory.rs#L463